content/pages/contact/speaker-bureau.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Speaker Bureau
+permalink: /speaker-bureau/
```

Comment:
concern (content): Are people familiar with the term "speaker bureau"? That's new to me.

change (content, suggested, non-blocking): Change the `permalink` to /speakers/

change (content, suggested, non-blocking): Change the `title` to something like "Have 18F speak at your event"